apps/src/standaloneVideo/StandaloneVideo.tsx

```diff
@@ -20,9 +20,22 @@ import styles from './video.module.scss';
 
 const StandaloneVideo: React.FunctionComponent = () => {
   const dispatch = useAppDispatch();
-  const levelVideo: VideoLevelData = useSelector(
+  const levelData = useSelector(
     (state: {lab: LabState}) => state.lab.levelData
-  ) as VideoLevelData;
+  );
+  const currentAppName = useSelector(
+    (state: {lab: LabState}) => state.lab.appName
+  );
+
+  const [levelVideo, setLevelVideo] = React.useState<VideoLevelData | null>(
+    null
+  );
+
+  useEffect(() => {
```

Comment:
Note: these changes aren't strictly necessary, as StandaloneVideo is not rendered in the background, so it's a relatively safe assumption that when rendered, the level data in Redux will always be Video level data. However, adding these checks here helps future-proof this a bit, in case we want to experiment with keeping Standalone Video in the background, and also gives an example of how labs can process these updates in a functional component.